### PR TITLE
Added docs on installing from source

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
Add `install from source` in get-started.md and fix some expressions.

### CODE REVIEW

File: docs/get-started.md [hunk 1]
```diff
@@ -116,7 +105,145 @@ Execution succeeded (Time spent: 154/793 us)
...
+
+Configure nebula-metad.conf
+
+In your Nebula installation directory, run
+
+```
+cp etc/nebula-metad.conf.default etc/nebula-metad.conf
+```
+
+Modify configurations in nebula-metad.conf, such as
```
Comment (whitewum) [inline docs/get-started.md]: @jude-zhu help to give the defalut value. ( or port ranges)

File: docs/get-started.md [hunk 2]
```diff
@@ -116,7 +105,145 @@ Execution succeeded (Time spent: 154/793 us)
...
+- ws_h2_port metaservice HTTP2
+
+
+Configure nebula-storaged.conf
+
+```
+cp etc/nebula-storaged.conf.default etc/nebula-storaged.conf
+```
+
+Modify configurations in nebula-storaged.conf, such as
```
Comment (whitewum) [inline docs/get-started.md]: @jude-zhu help to check:
if one begginer user only need to modify these four items, make it clear.  DO not use 'such as'. 'such as' mean more there are more than four items to modiry

File: docs/get-started.md [hunk 3]
```diff
@@ -116,7 +105,145 @@ Execution succeeded (Time spent: 154/793 us)
...
+- ws_h2_port metaservice HTTP2
+
+Configure nebula-graphd.conf
+
+```
+cp etc/nebula-graphd.conf.default etc/nebula-graphd.conf
+```
+
+Modify configurations in nebula-storaged.conf, such as
+
```
Comment (whitewum) [inline docs/get-started.md]: @darionyaphet by the way, does the configure file support comment (#).
we can add some examples by comment. 
User can freely and directly un-commment the working lines. (like ngnix configure file)

File: docs/get-started.md [hunk 4]
```diff
@@ -116,7 +105,145 @@ Execution succeeded (Time spent: 154/793 us)
...
+script/nebula.service status all
+```
+
+**Connect to Nebula**
+
+```
+bin/nebula -u=user -p=password
+```
+
+- -u is user name, default user
```
Comment (whitewum) [inline docs/get-started.md]: -u to set user name. `user` is a default nebula user account.

File: docs/get-started.md [hunk 5]
```diff
@@ -116,7 +105,145 @@ Execution succeeded (Time spent: 154/793 us)
...
+- -p is password, default password
```
Comment (whitewum) [inline docs/get-started.md]: -p is to give password. `password` is the default password for account `user`

File: docs/get-started.md [hunk 6]
```diff
@@ -116,7 +105,145 @@ Execution succeeded (Time spent: 154/793 us)
...
+
+Add host after connection:
```
Comment (whitewum) [inline docs/get-started.md]: to make console connect to the nebula service. @jude-zhu

File: docs/get-started.md [hunk 7]
```diff
@@ -116,7 +105,145 @@ Execution succeeded (Time spent: 154/793 us)
...
+
+```
+bin/nebula -u=user -p=password
+> ADD HOSTS $storage_ip:$storage_port
```
Comment (whitewum) [inline docs/get-started.md]: is graph config, or storage config ? @jude-zhu

File: docs/get-started.md [hunk 8]
```diff
@@ -116,7 +105,145 @@ Execution succeeded (Time spent: 154/793 us)
...
+```
+
+Replace the `$storage_ip` and `$storage_port` here according to the `local_ip`
+and `port` in nebula-storaged.conf
+
+Then you’re now ready to start using Nebula Graph.
+
+---
+
+## Update Nebula Graph
```
Comment (whitewum) [inline docs/get-started.md]: I think the rephase is "re-install Nebula graph".

File: docs/get-started.md [hunk 9]
```diff
@@ -116,7 +105,145 @@ Execution succeeded (Time spent: 154/793 us)
...
+
+Upgrading Nebula Graph is as easy as downloading and replacing the executable you’ve placed in your PATH, if you are using Docker, run docker upgrade nebula.
```
Comment (whitewum) [inline docs/get-started.md]: @jude-zhu stop all the services (try the command).
re-install by binary, ...
For docker, re-pull the lastest images:
docker pull vesoft/nebula-graph:latest

File: docs/get-started.md [hunk 10]
```diff
@@ -1,47 +1,33 @@
...
 ```
 
-If Docker is not found, please see [here](https://docs.docker.com/install/) for more information to install Docker.
+If `docker` is not found, please see [here](https://docs.docker.com/install/) for more information to install docker.
 
 After that, using
 ```
 docker pull vesoft/nebula-graph:latest
 ```
 to get `nebula` image and `docker images` that can display images status.
```
[the remark this answers is not in the document]
Comment (Amber1990Zhang) [reply docs/get-started.md]: fixed

File: docs/get-started.md (hunk 5 above)
Comment (Amber1990Zhang) [reply docs/get-started.md]: I think it should be "-p is to set password" here

File: docs/get-started.md (hunk 1 above)
Comment (jude-zhu) [reply docs/get-started.md]: in etc/nebula-metad.conf the default port is given.

Comment (jude-zhu) [reply docs/get-started.md]: Actually the default values have been set in etc/nebula-metad.conf

File: docs/get-started.md (hunk 7 above)
Comment (jude-zhu) [reply docs/get-started.md]: "add hosts" is to add storage host

File: docs/get-started.md [hunk 11]
```diff
@@ -116,7 +123,144 @@ Execution succeeded (Time spent: 154/793 us)
...
+
+---
+
+## Update Nebula Graph
+
+Upgrading Nebula Graph is as easy as downloading and replacing the executable you’ve placed in your PATH, if you are using Docker, run docker to upgrade nebula.
+
+---
+
+## Simple Query Demo
```
Comment (jude-zhu) [inline docs/get-started.md]: According to general usage, we first create space, insert data. And then query.

File: docs/get-started.md [hunk 12]
```diff
@@ -233,34 +377,34 @@ You can create both tag and edge's schema:
...
 // Insert some vertices: players and teams:
-INSERT VERTEX player(name, age) VALUES -8379929135833483044:("Amar'e Stoudemire", 36)
+nebula> INSERT VERTEX player(name, age) VALUES -8379929135833483044:("Amar'e Stoudemire", 36)
 
-INSERT VERTEX team(name) VALUES -9110170398241263635:("Magic")
+nebula> INSERT VERTEX team(name) VALUES -9110170398241263635:("Magic")
 
 // Insert some edges: likes and serves:
-INSERT EDGE like(likeness) VALUES -8379929135833483044 -> 6663720087669302163:(90)
+nebula> INSERT EDGE like(likeness) VALUES -8379929135833483044 -> 6663720087669302163:(90)
```
Comment (jude-zhu) [inline docs/get-started.md]: According to the steps, there is no node whose vertexId is 6663720087669302163. When insert the edge, the users may get confused.

File: docs/get-started.md [hunk 13]
```diff
@@ -158,22 +158,22 @@ Currently, we are using `git-lfs` to store the 3rd-party libraries so make sure
 **Compiling**
```
Comment (whitewum) [inline docs/get-started.md]: a tiny error in line 73: 
you're in the `root/` directory and you should use `cd ~/nebula-graph/`  to switch
you're in the root directory '/' and type `cd ~/nebula-graph/` to switch

File: docs/get-started.md (hunk 12 above)
Comment (whitewum) [reply docs/get-started.md]: > According to the steps, there is no node whose vertexId is 6663720087669302163. When insert the edge, the users may get confused.

right. I think it's better to simplify this example as following:

nebula> INSERT VERTEX player(name, age) VALUES 100:("Amar'e Stoudemire", 36)

nebula> INSERT VERTEX team(name) VALUES 200::("Magic")

nebula> INSERT EDGE like(likeness) VALUES 100 -> 200:(90)

**Comment by dutor** [comment]
Please resove the conflicts.

**Comment by whitewum** [comment]
please resolve the conflicts and commit again:
git add $filename.

**Comment by Amber1990Zhang** [comment]
> please resolve the conflicts and commit again:
> git add $filename.

Conflicts fixed, please review.
@whitewum @jude-zhu @dutor

File: docs/get-started.md [hunk 14]
```diff
@@ -1,57 +1,90 @@
...
 
 ```
-docker run -it vesoft/nebula-graph:latest /bin/bash
+> docker run -it vesoft/nebula-graph:latest /bin/bash
 ```
 
-to start and log in to the Docker container.
-After login, you're in the `root` directory and you should use `cd ~/nebula-graph/` to switch to the nebula home directory.
+to start and login to the docker container.
+After login, you're in the `root/` directory and you should use `cd ~/nebula-graph/` to switch to the nebula home directory.
```
Comment (whitewum) [inline docs/get-started.md]: you're in the root directory '/'

File: docs/get-started.md [hunk 15]
```diff
@@ -168,20 +344,27 @@ To show the attributes of a tag or an edge type:
...
-#### Insert Data
 
-Insert the vertexes and edges based on the graph above.
+### Insert Data
+
+
+Insert the vertices and edges based on the graph above.
 ```
--- Insert vertexes
+<<<<<<< HEAD
```
Comment (whitewum) [inline docs/get-started.md]: why conflicts?

File: docs/get-started.md [hunk 16]
```diff
@@ -226,9 +409,10 @@ nebula> GO FROM 201 OVER like WHERE $$[student].age >= 17 YIELD $$[student].name
 
 `$$` indicates the target vertex.
 
-Q3. Find the courses that the vetexes liked by 201 select and their grade.
+Q3. Find the courses that the vertices liked by 201 select and their grade.
 
 ```
+<<<<<<< HEAD
```
Comment (whitewum) [inline docs/get-started.md]: ditto

**Pull request merged**